package.json

```diff
@@ -136,5 +139,14 @@
     "test:command-reference": "./bin/dev commandreference:generate -p @salesforce/plugin-templates --erroronwarnings",
     "test:deprecation-policy": "./bin/dev snapshot:compare",
     "version": "oclif readme && git add README.md"
+  },
+  "hooks": {
```

Comment:
We've removed our husky config from the `package.json` and moved them to a `.husky` directory. [Example here](https://github.com/salesforcecli/plugin-template-sf/tree/main/.husky)